charts/ckan/templates/ingress.yaml

```
@@ -0,0 +1,47 @@
+{{- if .Values.ckan.ingress.hostname -}}
+{{- $fullName := include "common.names.fullname" . -}}
+{{- $svcName := printf "%s-%s" (include "common.names.fullname" .) "ckan"  -}}
+apiVersion: {{ include "common.capabilities.ingress.apiVersion" . }}
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    {{- include "common.labels.standard" . | nindent 4 }}
+  annotations:
+    {{- if .Values.ckan.ingress.certManager }}
+    kubernetes.io/tls-acme: "true"
+    {{- end }}
+    {{- if or .Values.ckan.ingress.annotations .Values.commonAnnotations }}
+    {{- $annotations := include "common.tplvalues.merge" ( dict "values" ( list .Values.ckan.ingress.annotations .Values.commonAnnotations ) "context" . ) }}
+    {{- include "common.tplvalues.render" ( dict "value" $annotations "context" $) | nindent 4 }}
+    {{- end }}
+spec:
+  {{- if and .Values.ckan.ingress.ingressClassName (eq "true" (include "common.ingress.supportsIngressClassname" .)) }}
+  ingressClassName: {{ .Values.ckan.ingress.ingressClassName | quote }}
+  {{- end }}
+  rules:
+    {{- if .Values.ckan.ingress.hostname }}
+    - host: {{ required "You must provide a host, when enabling ingress" .Values.ckan.ingress.hostname | quote }}
+      http:
+        paths:
+          - path: /
+            pathType: Prefix
+            backend: {{- include "common.ingress.backend" (dict "serviceName" $svcName "servicePort" "http" "context" $) | nindent 14 }}
+    {{- end }}
+  {{- if or (and .Values.ckan.ingress.tls (or (include "common.ingress.certManagerRequest" ( dict "annotations" .Values.ckan.ingress.annotations )) .Values.ckan.ingress.selfSigned)) .Values.ckan.ingress.extraTls }}
```

```suggestion
  {{- if .Values.ckan.ingress.tls }}
```

What is this `extraTls` or this `selfSigned` for?

---

i had stuck to the per se standard of bitnami etc. but removed it